src/ResourceManager/Insights/Commands.Insights.Test/Utilities.cs

```diff
@@ -236,7 +235,7 @@ public static void ExecuteVerifications(LogsCmdletBase cmdlet, Mock<IActivityLog
             VerifyFilterIsUsable(filter: filter);
             VerifyStartDateInFilter(filter: filter, startDate: null);
             VerifyConditionInFilter(filter: filter, field: requiredFieldName, value: requiredFieldValue);
-            VerifyContinuationToken(nextLink: nextLink);
+            //// VerifyContinuationToken(nextLink: nextLink);
```

Comment:
@pkamath13 @gucalder nit: remove the commented out lines in this file

Reply:
Removed.